espressostreamer/espresso_streamer.go

```diff
@@ -413,7 +481,13 @@ func (s *EspressoStreamer) Start(ctxIn context.Context) error {
 		} else {
 			log.Debug("Now processing hotshot block", "block number", s.nextHotshotBlockNum)
 		}
-		err := s.QueueMessagesFromHotshot(ctx, s.parseEspressoTransaction)
+		var err error
+		if s.isTimeboosted {
```

Comment:
instead of doing it this way, our espresso payload should have some bytes indicative of what type of payload is coming is it Timeboost or not. This way we can support multiple versions. 
We need to discuss this more on Slack on how we want to do this, I know Theo also wanted to add some header to the Espresso payload